tests/test_generator.py

```diff
@@ -46,7 +46,9 @@ def test_pattern_string():
         "pattern": "^(\\([0-9]{3}\\))?[0-9]{3}-[0-9]{4}$"
     }
     output = generate_json(schema)
+    match = re.fullmatch(schema.get("pattern"), output)
     print('result: ', output)
+    print('pattern match: Yes' if match != None else 'pattern match: No')
```

Comment:
Printing is helpful during local development, but we shouldn't push `print()` statements. Instead `assert` what we want to be true so that `pytest tests/` passes only when we're doing the right things.

Reply:
I added an assertion, should I delete any print statements in the test file?